plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/CoderWorkspacesCard.test.tsx

```diff
@@ -73,7 +73,7 @@ describe(`${CoderWorkspacesCard.name}`, () => {
 
         await user.tripleClick(searchbox);
         await user.keyboard('[Backspace]');
-        await user.keyboard('I can do it - I can do it nine times');
+        await user.keyboard('I-can-do-it-I-can-do-it-nine-times');
```

Comment:
Removed the spaces (1) to account for the mock server logic better, and (2) to account for the fact that the actual Coder API doesn't expect spaces like this, either